pipeline/batch.go

```diff
@@ -178,6 +182,21 @@ func (n *QueryNode) ChainMethods() map[string]reflect.Value {
 //
 // It is recommended to use QueryNode.Align and QueryNode.Offset in conjunction with
 // group by time dimensions so that the time bounds match up with the group by intervals.
+// To automatically align the group by intervals to the start of the query time,
+// use QueryNode.AlignGroup. This is useful in more complex situations, such as when
+// the groupBy period is larger than the query interval.
```

Comment:
As per your example can you change this to read `such as when the groupBy time period is longer than the query frequency.`

And maybe add something below the TICKscript example about how for every other query the groupBy time bounds would not be aligned (the ones ending on 30s instead of 0s) without the `.alignGroup` call.

Thanks